Sources/Support/Views/Internal/BulletinCloseButton.swift

```diff
@@ -121,7 +121,7 @@ class BulletinCloseButton: UIControl, HighlighterTarget {
 
 extension Bundle {
     fileprivate static var UIKitCore: Bundle {
-        return Bundle(identifier: "com.apple.UIKitCore")!
+        return Bundle(for: UIApplication.self)
```

Comment:
Could you update this to still use UIKitCore on iOS 12+?

```suggestion
        if #available(iOS 12, *) {
            return Bundle(identifier: "com.apple.UIKitCore")!
        } else {
            return Bundle(for: UIApplication.self)
        }
```

Reply:
Did this work? Seems like a significant issue